test/unit/acl/access-controller.test.ts

```diff
@@ -0,0 +1,89 @@
+import * as RdfLib from 'rdflib'
```

Comment:
Hmm, maybe make sure that you actually get rdflib here, rather than the mocks (Jest hoists `jest.mock` calls to the top, I believe). Otherwise that will probably be a surprise in the future.

Otherwise, you could look at using `require.requireActual` to get the unmocked version.

Reply:
We never want real rdflib or real s-a-c to be used, right?

Reply:
as discussed :)